source/core/module_coreHelpers.js

```diff
@@ -56,17 +56,21 @@
             fpSync = fn;
         };
 
-        Module.coreHelpers.utf8ArrayToStringWithNull = function (u8Array, idxIn) {
+        // Takes Vireo Strings that are UTF-8 encoded strings with known length and returns a JS string
+        // TODO mraj assumes valid UTF8 encoding https://github.com/ni/VireoSDK/issues/283
```

Comment:
I think we need to step back and come up with a strategy for how to handle incorrectly encoded UTF-8 strings in Vireo.

Just erroring out might not be the best approach because LV semantics for strings just treat them as byte arrays. So if a user makes a wacky string that is invalid utf8 and wires it to a string indicator that shouldn't throw an exception.